src/content/blog/2022-08-05-new-config-system-part-2.md

```diff
@@ -0,0 +1,512 @@
+
+1. You can now insert the parser object directly into the config.
+1. Parsers can now be bundled with plugins and you can specify a string value for `parser` to use a parser from a plugin. (Described more in the next section.)
+
+Here's an example using the [Babel ESLint parser](https://npmjs.com/package/@babel/eslint-parser):
+
+```js
+import babelParser from "@babel/eslint-parser";
+
+export default [
+    {
+        files: ["**/*.js", "**/*.mjs"],
+        languageOptions: {
+            parser: babelParser
+        }
+    }
+];
+```
+
+This configuration ensures that the Babel parser, rather than the default, will be used to parse all files ending with `.js` and `.mjs`.
+
+You can also pass options directly to the custom parser by using the `parserOptions` key in the same way as it works in eslintrc:
+
+```js
+import babelParser from "@babel/eslint-parser";
+
+export default [
+    {
+        files: ["**/*.js", "**/*.mjs"],
+        languageOptions: {
+            parser: babelParser,
+            parserOptions: {
+                requireConfigFile: false,
+                babelOptions: {
+                    babelrc: false,
+                    configFile: false,
+                    // your babel options
+                    presets: ["@babel/preset-env"],
+                }
+            }
+        }
+    }
+];
+```
+
+## More powerful and configurable plugins
+
+The strength of ESLint is the ecosystem of plugins that individuals and companies maintain to customize their linting strategy. As such, we wanted to be sure that existing plugins continued to work without modification as well as allowing plugins to do things they were never able to do in the past.
+
+On the surface, using a plugin in flat config looks very similar to using a plugin in eslintrc. The big difference is that eslintrc used strings whereas flat configs uses objects. Instead of specifying the name of a plugin, you import the plugin directly and place it into the `plugins` key, as in this example:
+
+```js
+import jsdoc from "eslint-plugin-jsdoc";
+
+export default [
+    {
+        files: ["**/*.js"],
+        plugins: {
+            jsdoc
+        }
+        rules: {
+            "jsdoc/require-description": "error",
+            "jsdoc/check-values": "error"
+        }  
+    }
+];
+```
+
+This config uses the [`eslint-plugin-jsdoc`](https://npmjs.com/package/eslint-plugin-jsdoc) plugin by importing it as a local `jsdoc` variable and then inserting it into the `plugins` key in the config. After that, the rules inside the plugin are referenced using the `jsdoc` namespace.
+
+**Note:** Because plugins are now imported like any other JavaScript module, there's no more strict enforcement of plugin package names. You no longer need to include `eslint-plugin-` as the prefix for your package names...but we would like it if you did.
```

Comment:
```suggestion
**Note:** Because plugins are now imported like any other JavaScript module, there's no more strict enforcement of plugin package names. You no longer need to include `eslint-plugin-` as the prefix for your package names... but we would like it if you did.
```

(Just a personal writing style suggestion-- I know ellipsis without a space is mostly acceptable today.)